infra/base-images/base-builder/compile

```diff
@@ -29,9 +29,17 @@ fi
 
 if [[ $SANITIZER_FLAGS = *sanitize=memory* ]]
 then
-  # Take all libraries from lib/msan
+  # Take all libraries from lib/msan and MSAN_LIBS_PATH
   # export CXXFLAGS_EXTRA="-L/usr/msan/lib $CXXFLAGS_EXTRA"
   cp -R /usr/msan/lib/* /usr/lib/
+
+  if [[ -z "${MSAN_LIBS_PATH-}" ]]; then
+    echo 'WARNING: Building without MSan instrumented libraries.'
+  else
+    # Copy all static libraries only. Don't include .so files because they can
+    # break non MSan compiled programs.
+    (cd "$MSAN_LIBS_PATH" && find . -name '*.a' -exec cp --parents '{}' / ';')
```

Comment:
where do we copy these to?

Reply:
To /. These libraries are e.g. /usr/lib/libz.a etc

Reply:
Ah, so there is a relative path! Thanks.